worlds/banjo_tooie/__init__.py

```diff
@@ -588,76 +588,78 @@ def validate_yaml_options(self) -> None:
         if self.options.randomize_worlds.value \
                 and self.options.randomize_bk_moves.value != RandomizeBKMoveList.option_none\
                 and self.options.logic_type.value == LogicType.option_intended:
-            raise OptionError("Randomize Worlds and Randomize BK Moves is not compatible with Intended Logic.")
+            raise OptionError("Banjo-Tooie: Randomize Worlds and Randomize BK Moves is not compatible with Intended Logic.")
         if not self.options.randomize_notes.value \
                 and not self.options.randomize_signposts.value and not self.options.nestsanity.value \
                 and self.options.randomize_bk_moves.value != RandomizeBKMoveList.option_none:
-            if self.multiworld.players == 1:
-                raise OptionError("Randomize Notes, signposts or nestsanity is required for Randomize BK Moves.")
+                raise OptionError(
+                    "Banjo-Tooie: Your options are too restrictive for Randomize BK Moves. \
+                    Try randomizing notes, signs or nestsanity"
+                )
```

Comment:
```suggestion
            raise OptionError(
                "Banjo-Tooie: Your options are too restrictive for Randomize BK Moves. "
                "Try randomizing notes, signs or nestsanity"
            )
```
This adds another place where there would be extra spaces included in the string from how it's written (which by the way, the last one seems to not have actually been applied here: https://github.com/ArchipelagoMW/Archipelago/pull/4471#discussion_r2644804841). Also if the if is removed, it'd be good to dedent this.

Although I'm not really sure that adding this to the messages is itself really necessary. Errors will already show which yaml they game from in the exception (unlike warnings), and I haven't seen other worlds do this generally.